libs/expo/betterangels/src/lib/screens/FileScreenComponent/index.tsx

```diff
@@ -111,17 +162,48 @@ export default function FileScreenComponent(props: TFileScreenComponent) {
             />
           )}
 
-          <TextBold mt="sm" size="sm">
-            File Name
-          </TextBold>
-          <TextRegular size="sm" style={{ width: '100%' }}>
-            {originalFilename}
-          </TextRegular>
+          {editing === 'true' ? (
```

Comment:
I don't love this str comparison but wasn't sure if there's a better standard approach. I considered using a variant but that would still result in a str comparison so 🤷